docs/v210/visa_eligibility_details.md

```diff
@@ -6,10 +6,62 @@ sass:
 
 # Canada Visa Eligibility Checks
 
-This release introduces a first iteration of functionality to support comprehensive Canada visa eligibility checks from
-within the TC, encompassing initial checks, job-specific questions, and standardized final assessment criteria. This 
-marks the first iteration of this functionality, which will further integrated in v2.2.0 with plans to extend to other 
-regions.
 
-<Link to Caroline's video's>
+This release introduces an updated iteration of visa eligibility checks, which was previously designed and used for 
+Australian visa checks only. The new functionality has been created to support the comprehensive Canada visa 
+eligibility check, but has been designed with all other destinations in mind. This will allow us to perform an 
+efficient rollout of this visa eligibility check functionality across all destinations in future releases of the TC.
+
+The functionality allows users to perform visa eligibility checks from within the TC, encompassing initial checks, 
+job-specific questions, and standardized final assessment criteria. Conducting visa eligibility checks on the TC allows 
+for easy access of the data that is required to effectively perform the check. It displays data from the intakes,
+candidate registration, job opportunity intake or any other useful data that exists on the TC. Each destination will 
+have their own set of questions and data to be displayed, however the overall design and functionality will be 
+standardised (as per what is shown in this release).
+
+
+# Section 1 - General Visa Assessment Questions
+
+The first section of the visa eligibility check collects data that is relating to the **candidate** and the 
+**destination country**. Keeping these questions in their own section allows them to be answered once and avoids 
+unnecessary data duplication.
+
+<div class="card-image-container">
+  <img src="./../assets/images/v210/CanadaVisaCheckSectionOne.png" alt="Enhanced Other Languages Search" class="card-image">
+</div>
+
+# Section 2 - Job Specific Assessment Questions
+
+The second section of the visa eligibility check collects data that will change depending on the particular job 
+opportunity. Data collected is job specific, such as relevant candidate experience for the role and 
+candidate's interest in the role etc. When adding a job to the candidate's visa eligibility check it is automated to 
+only show the candidate's job opportunities as selections to streamline the process and connect the associated job's 
+details to the visa check.
```

Comment:
I think that this sentence is a bit confusing. How about " When adding a job to the candidate's visa eligibility check only that candidate's job opportunities are presented for selection."

Reply:
Eep yes that sentence got lost somewhere! Thanks I've changed as you suggested.